guides/keras_nlp/getting_started.py

```diff
@@ -0,0 +1,619 @@
+"""
+Title: Getting Started with KerasNLP
+Author: [jbischof](https://github.com/jbischof)
+Date created: 2022-12-15
+Last modified: 2022-12-15
+Description: An introduction to the KerasNLP API.
+"""
+
+import keras_nlp
+import tensorflow as tf
+from tensorflow import keras
+
+# Use mixed precision for optimal performance
+keras.mixed_precision.set_global_policy("mixed_float16")
+
+"""
+## KerasNLP: Modular NLP Workflows for Keras
+
+
+KerasNLP is a natural language processing library that supports users through their
+entire development cycle. Our workflows are built from modular components that have SoTA
+preset weights and architectures when used out-of-the-box and are easily customizable
+when more control is needed.
+
+This library is an extension of the core Keras API; all high level modules are
+[`Layers`](https://keras.io/api/layers/) or
+[`Models`](https://keras.io/api/models/). If you are familiar with Keras,
+congratulations! You already understand most of KerasNLP.
+
+This guide demonstrates our modular approach using a sentiment analysis example at six
+levels of complexity:
+
+* Inference with a pretrained classifier
+* Fine tuning a pretrained backbone
+* Fine tuning with user-controlled preprocessing
+* Fine tuning a custom model
+* Pretraining a backbone model
+* Build and train your own transformer from scratch
+
+Throughout our guide we use Professor Keras, the official Keras mascot, as a visual
+reference for the complexity of the material:
+
+![picture](https://drive.google.com/uc?id=1d14Qpmfgjf6zu4z30HBaonH8PYDHgVoU)
```

Comment:
Before we commit, we should download these in `theme/img/` and reference them here. We probably don't want everyone on keras.io hitting your google drive.

Reply:
I think they should be in `guides/img/` right?